src/hevm/src/EVM/SMT.hs

```diff
@@ -740,6 +740,9 @@ withSolvers solver count cont = do
                   mempty (calldataV cexvars)
               pure $ Sat $ SMTCex
                 { calldata = calldatamodels
+                , storage = mempty
```

Comment:
these missing fields were triggering a runtime exception if we got `sat` back.

Reply:
Sorry, my mistake.